src/frontend/components/Standings/Relative.tsx

```diff
@@ -27,7 +28,39 @@ export const Relative = () => {
     // If no player found, return empty rows
     if (playerIndex === -1) {
       return Array.from({ length: totalRows }, (_, index) => (
-        <DummyDriverRow key={`empty-${index}`} />
+        <DriverInfoRow
+          key={`empty-${index}`}
+          carIdx={0}
+          classColor={0}
+          name="Franz Hermann"
+          isPlayer={false}
+          hasFastestTime={false}
+          hidden={true}
+          isMultiClass={false}
+          lapTimeDeltas={undefined}
+          numLapDeltasToShow={undefined}
+          displayOrder={settings?.displayOrder}
+          config={settings}
+          carNumber={settings?.carNumber?.enabled ?? true ? '' : undefined}
+          flairId={settings?.countryFlags?.enabled ?? true ? 0 : undefined}
+          carId={settings?.carManufacturer?.enabled ?? true ? 0 : undefined}
+          badge={settings?.badge?.enabled ? <></> : undefined}
+          iRating={settings?.iRating?.enabled ? undefined : undefined}
```

Comment:
@LanceVShoot what was the reason for adding iRating on its own? its part of the badge. If you wanted to hide it seperately we can just make it a setting for the badge itself.

Reply:
Yep i see it might've not been a "displayable" field. Probably hard to distinguish them hah. Thanks!